etna/alerts/models.py

```diff
@@ -72,11 +75,22 @@ class AlertMixin(models.Model):
         related_name="+",
     )
 
+    @property
+    def alert_check(self):
+        if self.alert and self.alert.active:
+            return self.alert
+        elif parent := self.get_parent():
+            if type(parent.specific) is not Page:
+                if parent_alert := parent.specific.alert_check:
+                    if parent_alert.cascade:
+                        return parent_alert
+        return None
+
```

Comment:
Do we want the "closest" alert to show (like it is currently)
e.g.
Page 1 has a child, Page 2. Page 1 has an alert on, Page 2 also has an alert on.
Page 2 has a child, Page 3. Should Page 2 and 3 take Page 1's alert? Or should Page 2 and 3 use Page 2's alert?

Or should we do it based on "alert level"?
e.g.
Page 1 has a "high" alert, Page 2 has a "medium" alert - should Page 2 use its own "medium" alert? Or should it take the highest level of alert (Page 1)

Reply:
Answer: We want the parent-most alert to show regardless of level.